src/commands/uplink/DataDayMul.ts

```diff
@@ -1,16 +1,12 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 import Command from '../../Command.js';
 import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
-import {getSecondsFromDate, getDateFromSeconds} from '../../utils/time.js';
+import {getSecondsFromDate} from '../../utils/time.js';
 import GetCurrentMul, {IGetCurrentMulParameters} from './GetCurrentMul.js';
 
 const COMMAND_ID = 0x16;
 const COMMAND_TITLE = 'DATA_DAY_MUL';
 const COMMAND_BODY_MAX_SIZE = 32;
```

Comment:
would be nice to explain this size as well

Reply:
description added